src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$sea-color: #00acdc;
+$header-logo-size: 40px;
+$dark-color: #060b35;
+$grey-color: #616070;
+$white-border-color: #f3f3f3;
+$white-color: #fff;
+$card-border-radius: 5px;
+$products-width: 944px;
+$star-size: 16px;
+$card-price-line: 18px;
+$card-width: 200px;
+$product-card-margin-right: 48px;
+$font-family-inherit: inherit;
```

Comment:
Do you really need this variable? Most of elements will inherit font-family by default